tests/integration-tests/tests/api/test_api_client.py

```diff
@@ -12,19 +12,28 @@
 
 
 import base64
+import logging
 
 import boto3
 import pytest
 from assertpy import assert_that, soft_assertions
 from botocore.config import Config
 from pcluster_client import ApiException
-from pcluster_client.api import cluster_operations_api
+from pcluster_client.api import cluster_operations_api, image_operations_api
+from pcluster_client.exceptions import NotFoundException
+from pcluster_client.model.build_image_request_content import BuildImageRequestContent
 from pcluster_client.model.cloud_formation_status import CloudFormationStatus
 from pcluster_client.model.cluster_status import ClusterStatus
 from pcluster_client.model.create_cluster_request_content import CreateClusterRequestContent
+from pcluster_client.model.image_build_status import ImageBuildStatus
+from pcluster_client.model.image_status_filtering_option import ImageStatusFilteringOption
 from pcluster_client.model.update_cluster_request_content import UpdateClusterRequestContent
 from utils import generate_stack_name
```

Comment:
shall we change this to import from tests.common.utils?

Reply:
I see that in all other tests (security groups, createami, etc.) we use this import. I tried changing it to `from tests.common.utils import generate_stack_name`, but with this last import in is not found.